CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -70,6 +74,8 @@ while not Player do
 	Player = PlayersService.LocalPlayer
 end
 
+local canChat = true
```

Comment:
We don't make that guarantee and we can't slow down the top bar setup waiting for this variable. We need to remove the top bar button when it is set.